### PR TITLE
Set up special circumstances page

### PR DESCRIPTION
**Pull request by cehsu**
These changes set up the special circumstances page.

There are some questions here regarding how this data should be stored (as none of the fields map to the backend schema), what conditional additional pages or fields this should present to the applicant (e.g. point of contact information), and whether the chapter name should reflect the page name (e.g. "Special Circumstances") as the applicant is not reviewing anything here.

image

### CODE REVIEW

**Comment by cvalarida** [comment]
@goldenmeanie My understanding is we may also be filling out ancillary forms with this. Does any of this information get recorded in an ancillary form?

File: test/disability-benefits/526EZ/config/specialCircumstances.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { mount } from 'enzyme';
+
+import { DefinitionTester } from '../../../../src/platform/testing/unit/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/disability-benefits/526EZ/config/form.js';
+
+describe('Disability benefits 526EZ special circumstances', () => {
+  const { schema, uiSchema } = formConfig.chapters.reviewVeteranDetails.pages.specialCircumstances;
+  it('renders evidence type form', () => {
```
Comment (cvalarida) [inline test/disability-benefits/526EZ/config/specialCircumstances.unit.spec.jsx]: Is this description right?

**Review by cvalarida — APPROVED** [review]
:rocket:

**Comment by goldenmeanie** [comment]
@cvalarida these flashes don't specifically relate to the ancillary forms.

**Comment by cvalarida** [comment]
@goldenmeanie Gotcha, okay. I also don't remember seeing any place in the 526 submit endpoint documentation for this information. Do you know what's supposed to happen with this? Do all these options lead to VCL?

**Review by peggygannon — COMMENTED** [review]
@cehsu Can you please make the following edits to this screen: 
1. please put **Are you:** in bold

2. update the homeless answer to: 
Homeless or at risk of becoming homeless?

**Comment by cehsu** [comment]
Thanks @peggygannon, updated: 
screen shot 2018-04-13 at 4 53 35 pm

File: test/disability-benefits/526EZ/config/specialCircumstances.unit.spec.jsx (hunk 1 above)
Comment (cehsu) [reply test/disability-benefits/526EZ/config/specialCircumstances.unit.spec.jsx]: Nice catch, fixed.

**Pull request merged**